lib/spring/test/application.rb

```diff
@@ -94,7 +94,7 @@ def run(command, opts = {})
         Bundler.with_clean_env do
           Process.spawn(
             env,
-            command.to_s,
+            "bundle exec #{command}",
```

Comment:
Why is this necessary? Which tests fail without it?

We are already setting `GEM_PATH` and `GEM_HOME` in the env of the spawned process, so I don't understand why it would be an issue.